# *******************************************************************************
# Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
# SPDX-License-Identifier: Apache-2.0
#
# Licensed under the Apache License, Version 2.0 (the "License");
# you may not use this file except in compliance with the License.
# You may obtain a copy of the License at
#
#     http://www.apache.org/licenses/LICENSE-2.0
#
# Unless required by applicable law or agreed to in writing, software
# distributed under the License is distributed on an "AS IS" BASIS,
# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
# See the License for the specific language governing permissions and
# limitations under the License.
# *******************************************************************************
diff --git a/build_torch_xla_libs.sh b/build_torch_xla_libs.sh
index 82b70559..d0313150 100755
--- a/build_torch_xla_libs.sh
+++ b/build_torch_xla_libs.sh
@@ -66,6 +66,10 @@ if [[ "$XLA_CUDA" == "1" ]]; then
   OPTS+=(--config=cuda)
 fi
 
+if [[ "$XLA_CPU_USE_ACL" == "1" ]]; then
+  OPTS+=("--define=build_with_acl=true")
+fi
+
 if [ "$CMD" == "clean" ]; then
   pushd $THIRD_PARTY_DIR/tensorflow
   bazel clean

